docs/conf/inspircd.conf.example

```diff
@@ -177,11 +177,13 @@
 # module).
 #<bind address="" port="7002" type="clients" hook="websocket">
 
-# When linking servers, the OpenSSL and GnuTLS implementations are completely
-# link-compatible and can be used alongside each other
-# on each end of the link without any significant issues.
-# Supported SSL types are: "openssl" and "gnutls".
-# You must load the ssl_openssl module for OpenSSL or ssl_gnutls for GnuTLS.
+# When linking servers, the OpenSSL, GnuTLS, and mbedTLS implementations are
+# completely link-compatible and can be used alongside each other on each end
+# of the link without any significant issues.
+# Supported SSL types are:
+#   "openssl" (requires the ssl_openssl module)
+#   "gnutls" (requires the ssl_gnutls module)
+#   "mbedtls" (requires the ssl_mbedtls module)
```

Comment:
This is not the case anymore, ssl profiles are given a name in their tag and referenced by that name in `<link>` or `<bind>`, for example:

    <sslprofile name="strong" provider="openssl" ciphers="foo" certfile="cert1.pem" keyfile="...">
    <sslprofile name="normal" provider="openssl" ciphers="bar" certfile="cert2.pem" keyfile="...">
    <sslprofile name="normal-websocket" provider="openssl" ciphers="baz" certfile="cert2.pem" keyfile="..." requestclientcert="false">
    <bind address="..." port="6697" type="clients" ssl="normal">
    <bind address="..." port="7002" type="clients" hook="websocket" ssl="normal-websocket">
    <bind address="..." port="7000,7001" type="servers" ssl="strong">

The `<gnutls>`, `<openssl>` and `<mbedtls>` tags are only used if there is no `<sslprofile>` tag in the config, to retain compatibility with configs from 2.0.